app/models/micropost.rb

```diff
@@ -1,2 +1,8 @@
 class Micropost < ApplicationRecord
+  belongs_to :user
+  validates :content, length: {
+    minimum: 3,
+    maximum: 120
```

Comment:
If I'm not mistaken, `maximum` was `140`.

Reply:
Edited